packages/roosterjs-editor-dom/lib/list/VList.ts

```diff
@@ -182,10 +182,18 @@ export default class VList {
         this.rootList.parentNode.replaceChild(placeholder, this.rootList);
 
         this.items.forEach(item => {
+            if (item.getNewListStart()) {
```

Comment:
please also check if this number is different than exist "start" number. And we only need to split when they are different.

Reply:
Then set the "start" number to be this value, so that existing code below can use it